Crash.ClientTestApp/Program.cs

```diff
@@ -21,7 +22,7 @@ static void Main(string[] args)
             Console.WriteLine("\tc = Create Speck");
             Console.WriteLine("\tq = Quit");
 
-            var client = new CrashClient(userId, new Uri("http://localhost:5141/Crash"));

```

Comment:
I found 5141 had some issues in cafes I worked in. Unsure if there is any issue with changing this as 8080 worked better. I have no strong opinions either way